kratos/utilities/timer.h

```diff
@@ -20,14 +20,12 @@
 #include <iostream>
 #include <fstream>
 #include <map>
-#include <ctime>
-
 #ifdef _OPENMP
 #include <omp.h>
 #endif
 
 // External includes
-#include <boost/timer.hpp> // to be removed after replacing the boost timers with Kratos timer.
+#include <boost/timer.hpp> // To be removed after replacing the boost timers with Kratos timer. Additionallty to be moved to the cpp
```

Comment:
same

Reply:
I cannot remove #include <boost/timer.hpp> because due to some lazy practices on Kratos code, people include the timer to later use the boost timer directly (instead of using the timer or importing the boost library directly). I  I remove it it breaks the compilation of some files. I propose to remove it later in a different PR